.github/workflows/testing-and-deployment.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v4
         with:
-          name: ${{ github.job }}-${{ echo toJSON(matrix) | jq -r 'to_entries[] | "\(.key)-\(.value)"' | tr '\n' '_' | sed 's/_$//' }}
+          name: ${{ github.job }}-${{ join(matrix.* , '-/') }}
```

Comment:
looks like the slash doesn't work, just use '-'?

Reply:
yea I don't know how the '/' got there in the first place